addons/l10n_syscohada/data/account.account.template.csv

```diff
@@ -999,66 +988,66 @@
+"pcg_849","REPRISES DE CHARGES POUR DEPRECIATIONS ET PROVISIONS POUR RISQUES A COURT TERME H.A.O.","849","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_851","DOTATIONS AUX PROVISIONS RÉGLEMENTÉES","851","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_852","DOTATIONS AUX AMORTISSEMENTS H.A.O.","852","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_853","DOTATIONS AUX DÉPRÉCIATION H.A.O.","853","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_854","DOTATIONS AUX PROVISIONS POUR RISQUES ET CHARGES H.A.O.","854","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_858","AUTRES DOTATIONS H.A.O.","858","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_861","REPRISES DE PROVISIONS REGLEMENTEESREPRISES DE PROVISIONS RÉGLEMENTÉES","861","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_862","REPRISES D’AMORTISSEMENTS H.A.OREPRISES D’AMORTISSEMENTS H.A.O","862","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_863","REPRISES DE DEPRECIATIONS H.A.O.REPRISES DE DÉPRÉCIATION H.A.O.","863","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_864","REPRISES DE PROVISIONS POUR RISQUES ET CHARGES H.A.O.","864","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_868","AUTRES REPRISES H.A.O.","868","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_871","PARTICIPATION LÉGALE AUX BÉNÉFICES","871","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_874","PARTICIPATION CONTRACTUELLE AUX BÉNÉFICES","874","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_878","AUTRES PARTICIPATIONS","878","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_881","ÉTAT","881","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_884","COLLECTIVITÉS PUBLIQUES","884","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_886","GROUPE","886","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_888","AUTRES","888","income","l10n_syscohada.syscohada_chart_template","False"
+"pcg_8911","Activités exercées dans l'État","8911","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_8912","Activités exercées dans les autres États de la Région","8912","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_8913","Activités exercées hors Région","8913","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_892","RAPPEL D'IMPÔTS SUR RÉSULTATS ANTÉRIEURS","892","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_895","IMPÔT MINIMUM FORFAITAIRE (I.M.F.)","895","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_8991","Dégrèvements","8991","expense","l10n_syscohada.syscohada_chart_template","False"
+"pcg_8994","Annulations pour pertes rétroactives","8994","expense","l10n_syscohada.syscohada_chart_template","False"
```

Comment:
Same for this?

> i have a pr to change that in V15 that wait this one to be merged

Why do you have to wait on this one? I think it would be clearer if the purpose of the PRs stayed split.
There is nothing in the commit message about these changes

Reply:
Those ones were misclassified, Yes i can reset those changes and use my other PR for the changes no problem, i will change that this afternoon

Reply:
I wanted to wait since the account_type changed in V16.0 but i can apply the script on the FW of the other